src/storage/disk_manager.c

```diff
@@ -5311,16 +5374,62 @@ disk_vhdr_get_vol_remarks (const DISK_VOLUME_HEADER * vhdr)
   return ((char *) (vhdr->var_fields + vhdr->offset_to_vol_remarks));
 }
 
+
+/*
+ * disk_vhdr_get_vol_fullname_size () - get vol_fullname_size from volume header
```

Comment:
function description을 작성할 것이라면, 이 함수가 무엇을 하는지 직관적으로 보이면 좋을 것 같습니다. 
volume header에는 vol_fullname_size 라는 변수는 존재하지 않습니다. 
```suggestion
 * disk_vhdr_get_vol_fullname_size () - get the size of the volume's fullname using the vhdr(in).
```

Reply:
해당 사항 반영하여 수정하도록 하겠습니다!